docs/managing-compute-resources.asciidoc

```diff
@@ -64,13 +64,13 @@ spec:
             cpu: 2
 ----
 
-<1> Replace with `kibana` or `apm-server` as appropriate.
+For `containers` name, you can use `apm-server` for `kibana` as appropriate.
```

Comment:
```suggestion
For `containers` name, you can use `apm-server` or `kibana` as appropriate.
```

I liked the marker because it was very obvious what needed to change. Now it's easy to miss and not very clear IMO.

Reply:
I see your point. However, it's a blue marker on a dark background, not immediately obvious. Let's try to write as "accessible" as possible.